Docs/developer_guide/extensions.md

```diff
@@ -142,6 +141,7 @@ Replace `Release` with the build mode of your extension build (`Debug`, `Release
 - Set the project as the StartUp Project (right-click -> Set As StartUp Project).
 - Start debugging (F5).
 
+=(extension-continuous-integration)
```

Comment:
I don't know this markdown syntax. What does this do?

Reply:
Since header can be changed, this allows to have an anchor.
See https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#creating-explicit-targets